tests/unit/bokeh/test_tile_providers.py

```diff
@@ -115,13 +130,77 @@ def test_copies(self, name):
         p2 = bt.get_provider(getattr(bt, name))
         assert p1 is not p2
 
+@pytest.mark.unit
```

Comment:
You need to add:
```py
@pytest.mark.parametrize('name', ['OSM''])
```
etc. as well. See other tests (e.g. line 110).

Reply:
ok, right - got it. I missed that - thanks. added

Reply:
> You need to add:
> 
> ```python
> @pytest.mark.parametrize('name', ['OSM'])
> ```
> 
> etc. as well. See other tests (e.g. line 110).

ok, right - got it - I missed that - thanks. added